tap-agent/src/agent/sender_allocation.rs

```diff
@@ -187,6 +187,20 @@ impl Actor for SenderAllocation {
             allocation_id = %state.allocation_id,
             "Closing SenderAllocation, triggering last rav",
         );
+        loop {
+            match state.refresh_all_database().await {
+                Ok(value) => {
+                    state.unaggregated_fees = value;
+                    break;
+                }
+                Err(err) => {
+                    error!(
+                        error = %err,
+                        "There was an error while calculating last unaggregated receipts. Retrying in 30 seconds...");
+                    tokio::time::sleep(Duration::from_secs(30)).await;
```

Comment:
If `state.refresh_all_database` errors, this will log the error. If `state.refresh_all_database` continues to error, won't we just get stuck in an infinite loop? There's no fallback functionality is there?

Reply:
No, we are closing the allocation. There's nothing else to do other than retry again. Same happens for last rav, we need to aggregate all pending fees so we can mark it as last.

We don't have any fallback other than human interaction.